test/EmbedIO.Tests/Issues/Issue355_ContentResponseLength.cs

```diff
@@ -7,16 +7,21 @@ namespace EmbedIO.Tests.Issues
 {
     public class Issue355_ContentResponseLength
     {
-        [Test]
-        public async Task ActionModule_Handle_ContentLengthProperly()
+        [TestCase(true)]
```

Comment:
Two separate tests would be better IMHO. `TestCaseAttribute` should be used to inject data in tests, not to change the behavior of a test.

Reply:
Done!